### ISSUE TITLE
Still ud?

### ISSUE THREAD

**Issue by Loobo1** [Jire/Charlatano#918]
Hey! i used this cheat about a year ago is it still UD?

**Comment by effingnoone** [comment]
Yes still UD. And I wish Jire could give some more updates and security features like randomizing the program name etc. Or atleast we can have a official forum or discord server)(I know Jire gets banned on discord) but there are many other options since this cheat has 100 thousand users atm.

**Comment by Loobo1** [comment]
> Yes still UD. And I wish Jire could give some more updates and security features like randomizing the program name etc. Or atleast we can have a official forum or discord server)(I know Jire gets banned on discord) but there are many other options since this cheat has 100 thousand users atm.

What do u think about makeing one ;) i like makeing ds servers

**Comment by Loobo1** [comment]
https://discord.gg/BEVyUj Feel free to join

**Comment by effingnoone** [comment]
It's no use if Jire himself isn't in it. Come one Jire u can be on discord in disguise.
